assets/js/productDependencies.js

```diff
@@ -0,0 +1,34 @@
+import mermaid from '/assets/mermaid.esm.min.mjs'
+
+jQuery(async function () {
+  mermaid.initialize({ startOnLoad: false })
+
+  const drawDiagram = async function () {
+    const element = document.querySelector('.dependency-graph')
+    const graphDefinition = document.getElementById('graph-source').innerText
+
+    const { svg } = await mermaid.render('graph-svg', graphDefinition)
+    element.innerHTML = svg.replace(/[ ]*max-width:[ 0-9\.]*px;/i, '')
+    const panZoomTiger = svgPanZoom('#graph-svg', {
+      zoomEnabled: true,
+      fit: true,
+      center: true,
+    })
+    document.getElementById('graph-svg').setAttribute('height', '1500px')
+  }
+  await drawDiagram()
```

Comment:
Does this need to be a function if it only happens once, can the code in the function not just run? 🤔